image/build_windows_containerdisk.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+set -e
+
+if [ -z "$1" ]; then
+  echo "Please pass Windows image version to build as first argument."
+  exit 1
+fi
+
+if [ -z "$2" ]; then
+  echo "Please pass path to Windows ISO as second argument."
+  exit 1
+fi
+
+WINDOWS_VERSION=$1
+WINDOWS_ISO=$2
+case "$WINDOWS_VERSION" in
+  windows10)
+    OS_VARIANT=win10
+    VIRT_INSTALL_EXTRA_ARGS=(--boot uefi)
+    ;;
+  windows11)
+    OS_VARIANT=win11
```

Comment:
I guess you don't use `VIRT_INSTALL_EXTRA_ARGS` here because uefi is the default one on w11, right?

Reply:
Right